src/pages/about.jsx

```diff
@@ -1,30 +1,84 @@
+import React from 'react';
 import { View, Text, Container, Heading, ConnectingShapes } from "@go1d/go1d";
 import mobileApp from "./mobile_app.png";
+import check from "./CheckCirclecheckmark.png";
 
 const About = () => {
   return (
     <Container
-      backgroundColor="dangerHigh"
+      backgroundColor="dangerMid"
```

Comment:
Is this the right colour to use that matches the design?

Reply:
yes.